backend/scraper/mapping_police_violence.py

```diff
@@ -63,12 +64,14 @@ def make_single_table(table, dat, configs):
         configs["tables"][table]["required"]
         + configs["tables"][table]["optional"]
     )
+
     return dat[dat.columns[dat.columns.isin(cols)]]
 
 
 def extract_all_subfolders(head_directory, data_source, xwalk, dat, configs):
     if os.path.isdir(head_directory):
         for filename in os.listdir(head_directory):
+            print(filename)
             if configs["sources"][data_source]["subdirectory"] not in filename\
                     and configs["sources"][data_source]["subdirectory"]\
                     != "None":
```

Comment:
Below, you have a magic `nrows=1048576` in two places. Why do we need to specify that? It would be applied to all data sources, but I'd guess diff sources have diff numbers of rows.

Reply:
We give it the maximum number of rows to ensure it has enough.